pacu/modules/iam__privesc_scan/main.py

```diff
@@ -424,7 +424,7 @@ def main(args, pacu_main: 'Main'):
                                         is_possible = False
                                         break
 
-                                elif permission not in user['Permissions']['Allow']:  # and the user doesn't have it allowed
+                                if permission not in user['Permissions']['Allow']:  # and the user doesn't have it allowed
```

Comment:
As a warning, the control flow of execution **_is changed_** do to change in the keyword statement. As long as the control flow matches you're expectation, nothing needs to be done.

Reply:
The statement above end with a break so this should only ever be reached if the statement is not true anyways.